kolibri/core/assets/src/views/k-textbox/index.vue

```diff
@@ -32,6 +31,9 @@
    */
   export default {
     name: 'k-textbox',
+    $trs: {
+      required: 'This field is required',
```

Comment:
I'm wondering if it might be better to remove the `required` prop entirely, and let the outer form handle it like other validation.

On one hand, the prop is very convenient shorthand because it's a common pattern. On the other hand, it makes the textbox more complex by adding a "special" validation path.

My thinking is that in the longer-term, it would be better to accumulate a set of helper validation functions that can be imported and re-used at the form level rather than the textbox level. They would be able to return specific messages depending on their content.

E.g.:

* `required('')` might return `This field is required`
* `validUsername('r')` might return `A username must have at least two characters`
* `validUsername('&&&&&')` might return `A username can only have letters and numbers`